### PR TITLE
DOC + PEP8: Mostly just line-wrapping.

### PR DESCRIPTION
**Pull request by arokem**
And a few small edits on the csd example.

### CODE REVIEW

File: doc/examples/reconst_csd.py [hunk 1]
```diff
@@ -70,8 +73,8 @@
 """
 0.21197
 
-We can double check that we have a good response function by visualizing the
-response's function's ODF. Here is how:
+We can double-check that we have a good response function by visualizing the
+response's function's ODF. Here is how you would do that:
```
Comment (Garyfallidis) [inline doc/examples/reconst_csd.py]: typo: response function's 
And I would remove all from Here ... 
I think it is redundant

File: doc/examples/reconst_csd.py [hunk 2]
```diff
@@ -98,18 +101,17 @@
 fvtk.rm(ren, response_actor)
 
 """
-However, using an FA threshold is not a very robust method. It is dependent on
-the dataset (non-informed used subjectivity), and still depends on the
-diffusion tensor (FA and first eigenvector), which has low accuracy at high
-b-value. Alternatively, one can calibrate the response function directly from
-the data according to [Tax2014]_. 
+Depending on the dataset, FA threshold may not be the best way to divine the
```
Comment (Garyfallidis) [inline doc/examples/reconst_csd.py]: divine?

Comment (arokem) [inline doc/examples/reconst_csd.py]: From the Oxford English Dictionary:


"Divine, v. : 2. To make out by sagacity, intuition, or fortunate
conjecture (that is, in some other way than by actual information); to
conjecture, guess."

Comment (Garyfallidis) [inline doc/examples/reconst_csd.py]: Keep it simple. This use of the word is very rare afaik. 
Even guess is too vague here. 

I would say: find the best possible response function.
Your call!

File: doc/examples/reconst_csd.py (hunk 1 above)
Comment (arokem) [inline doc/examples/reconst_csd.py]: Done

File: doc/examples/reconst_csd.py (hunk 2 above)
Comment (arokem) [inline doc/examples/reconst_csd.py]: done

**Pull request merged**